tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
```

Comment:
You should add strict: true if doing so creates no type errors or creates just 1 or 2.   Otherwise it's not worth the effort.   Reason being it's shocking to me how often I've ran into the situation where some tool I wanted to use only worked if strict mode was on.